python/hail/tests/test_expr.py

```diff
@@ -1152,3 +1152,88 @@ def test_interval_ops(self):
         self.assertTrue(li.overlaps(li4).value)
         self.assertFalse(li.overlaps(li3).value)
         self.assertFalse(li.overlaps(li5).value)
+
+    def test_mendel_error_code(self):
+        locus_auto = hl.Locus('2', 20000000)
+        locus_x_par = hl.get_reference('default').par[0].start
+        locus_x_nonpar = hl.Locus(locus_x_par.contig, locus_x_par.position - 1)
+        locus_y_nonpar = hl.Locus('Y', hl.get_reference('default').lengths['Y'] - 1)
+
+        self.assertTrue(hl.all(lambda x: x, hl.array([
+            hl.literal(locus_auto).in_autosome(),
+            hl.literal(locus_auto).in_autosome_or_par(),
+            ~hl.literal(locus_x_par).in_autosome(),
+            hl.literal(locus_x_par).in_autosome_or_par(),
+            ~hl.literal(locus_x_nonpar).in_autosome(),
```

Comment:
should this be `in_autosome_or_par()`?

Reply:
sure. this was mostly for my sanity to make sure I was writing valid loci